### PR TITLE
Profile for update_prs

### PR DESCRIPTION
**Pull request by viniciusdc**
Using just the basic config (and the function/process name to help inspections inside the `profile` file).

### CODE REVIEW

File: conda_forge_tick/update_prs.py [hunk 1]
```diff
@@ -188,6 +199,13 @@ def main(args: "CLIArgs") -> None:
         # This function needs to run last since it edits the actual pr json!
         gx = close_dirty_prs(gx, args.dry_run)
 
+    # stop profiler
+    prof.disable()
+
+    # output to data
+    os.makedirs("profiler", exist_ok=True)
+    prof.dump_stats(f"profiler/update_prs_{current_time}.")
```
Comment (CJ-Wright) [inline conda_forge_tick/update_prs.py:207]: can this go in a subfolder (and put the other in a subfolder)?

Comment (viniciusdc) [reply conda_forge_tick/update_prs.py:207]: sure, I will modify that. Thanks for the advice 😄

Comment (viniciusdc) [reply conda_forge_tick/update_prs.py]: done 😄

**Comment by CJ-Wright** [comment]
Sorry for the delay!

**Pull request merged**